Loop/Managers/LoopDataManager.swift

```diff
@@ -201,10 +201,11 @@ final class LoopDataManager {
         - recommendedTempBasal: The recommended temp basal based on predicted glucose
         - lastTempBasal:        The last set temp basal
         - lastLoopCompleted:    The last date at which a loop completed, from prediction to dose (if dosing is enabled)
+        - lastLoopError         The error, if any, from the last loop attempt
         - insulinOnBoard        Current insulin on board
         - error:                An error object explaining why the retrieval failed
      */
-    func getLoopStatus(resultsHandler: (predictedGlucose: [GlucoseValue]?, recommendedTempBasal: TempBasalRecommendation?, lastTempBasal: DoseEntry?, lastLoopCompleted: NSDate?, insulinOnBoard: InsulinValue?, error: ErrorType?) -> Void) {
+    func getLoopStatus(resultsHandler: (predictedGlucose: [GlucoseValue]?, recommendedTempBasal: TempBasalRecommendation?, lastTempBasal: DoseEntry?, lastLoopCompleted: NSDate?, lastLoopError: ErrorType?, insulinOnBoard: InsulinValue?, error: ErrorType?) -> Void) {
```

Comment:
Well, if enact fails, lastLoopError will be set, and update() will not throw the same error.  I suppose if update() succeeds, and lastLoopError is set, I could return that, and keep a single error argument.